_posts/2017-08-21-police-malpractice.md

```diff
@@ -8,33 +8,24 @@ tags: politics
 
-If you go to the gym, your health insurance gets cheaper. If you replace your brakes, your car insurance gets cheaper. If you fix your roof, your homeowner's insurance gets cheaper. The insurance company creates incentives for responsible long-term behavior, and it's not because they care about your health or safety. They make their money by identifying liabilities and putting a price tag on them.
-
-![Noodle's first protest](/assets/images/wide/blm-16x9.png)
+![Noodle's first protest](/assets/images/blm.png)
 *[George Floyd](https://en.wikipedia.org/wiki/Death_of_George_Floyd) was killed in police custody less than a mile from our home.*
 
-American police departments pay out [hundreds of millions of dollars](https://www.wsj.com/articles/cost-of-police-misconduct-cases-soars-in-big-u-s-cities-1437013834) each year due to police misconduct, including wrongful deaths at the hands of police[^2]. Clearly, getting liabilities under control could save taxpayers quite a bit of money. But that's the thing: because police departments are publicly funded, there are no shareholders to push back against reckless spending. Police may even piggyback on the [city's insurance](http://www.savmn.com/DocumentCenter/View/568/6-26-17--Joint-Statement-City-of-St-Anthony-and-Valerie-Castile-as-Trustee-for-Philando-Castile) policy, further insulating them from the costs they incur.
+American police departments pay out [hundreds of millions of dollars](https://www.wsj.com/articles/cost-of-police-misconduct-cases-soars-in-big-u-s-cities-1437013834) each year due to police misconduct, including wrongful deaths at the hands of police[^2]. Clearly, getting liabilities under control could save taxpayers quite a bit of money. But that's just the thing: because police departments are publicly funded, there are no shareholders to push back against reckless spending. Police may even piggyback on the [city's insurance](http://www.savmn.com/DocumentCenter/View/568/6-26-17--Joint-Statement-City-of-St-Anthony-and-Valerie-Castile-as-Trustee-for-Philando-Castile) policy, further insulating them from the costs they incur.
```

Comment:
> there are no shareholders to push back against reckless spending.

What about voters?

Reply:
"to push back in real time"?

voters don't have a direct financial stake in government efficiency, but maybe response time is clearer as a barrier to feedback